libs/application/template-api-modules/src/lib/modules/templates/inheritance-report/inheritance-report.service.ts

```diff
@@ -49,11 +49,11 @@ export class InheritanceReportService extends BaseTemplateApiService {
     return result
   }
 
-  async syslumennOnEntry({ application, auth }: TemplateApiModuleActionProps) {
+  async syslumennOnEntry({ application }: TemplateApiModuleActionProps) {
     const [relationOptions, inheritanceReportInfos] = await Promise.all([
       this.syslumennService.getEstateRelations(),
       // Get estate info from syslumenn or fakedata depending on application.applicant
-      application.applicant.startsWith('010130') &&
+      application.applicant.startsWith('110130') &&
```

Comment:
This change makes us use the testdata from DC but it is now standardized such that we can use it thoroughly